content/en/Community/_index.md

```diff
@@ -100,7 +107,7 @@ Code Owners are a special type of contributor and have _significantly_ contribut
 - Member of the kyverno channel on Kubernetes and/or CNCF Slack
 - Attended five (5) Contributors Meetings as documented
 - Registered for the Kyverno [mailing list](https://groups.google.com/g/kyverno)
-- Create a pull request to add self to `CODEOWNERS` file in at least one (1) repository
+- Create a pull request to add self to [CODEOWNERS](https://github.com/kyverno/kyverno/blob/main/CODEOWNERS) file in at least one (1) repository
```

Comment:
The reason we didn't link to a specific file is because of the clause "in at least one (1) repository." You're now pointing them to a specific one rather than letting them choose a repository. I suppose that's fine, but then the sentence should be updated.

Reply:
Thanks for pointing it out. Removed links.